scania-digital-design-system/tegel
# feat(textfield): correct colors.

**Describe pull-request**  
Addressing some incorrect colors in the Texfield component. 

**Solving issue**  
Fixes: [DTS-1981](https://tegel.atlassian.net/jira/software/projects/DTS/boards/1?assignee=63ff6d6c314f5088138187f0&selectedIssue=DTS-1981)

**How to test**  
1. Go to Components -> Textfield
2. Check in the different states, types, light/dark-mode, and compare with Figma

[DTS-1981]: https://tegel.atlassian.net/browse/DTS-1981?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
